apps/projects/forms.py

```diff
@@ -70,11 +71,21 @@ def clean(self):
         return cleaned_data
 
 
-class PointForm(ProjectDashboardForm):
+class PointForm(PointFormMixin, ProjectDashboardForm):
     class Meta:
         model = Project
-        fields = ["administrative_district", "point"]
+        geo_field = "point"
+        fields = [
+            "administrative_district",
+            "point",
+            "street_name",
+            "house_number",
+            "zip_code",
+        ]
         required_for_project_publish = []
         widgets = {
             "point": maps_widgets.MapChoosePointWidget(polygon=settings.BERLIN_POLYGON),
         }
+
+        def get_geojson_properties(self):
+            return {"strname": "street_name", "hsnr": "house_number", "plz": "zip_code"}
```

Comment:
@partizipation @vellip forms that use the DashboardForm with a map also need these extra fields.